src/ChannelService.php

```diff
@@ -105,4 +107,113 @@ public function getById($serviceId, $channelId)
 
         return $channel;
     }
+
+    /**
+     * Get the Open now status as Value object.
+     *
+     * @param int $serviceId
+     *   The Service ID.
+     * @param int $channelId
+     *   The Channel ID.
+     *
+     * @return \StadGent\Services\OpeningHours\Value\OpenNow
+     *
+     * @throws \Exception
+     * @throws \GuzzleHttp\Exception\RequestException
+     * @throws \Psr\SimpleCache\InvalidArgumentException
+     * @throws \StadGent\Services\OpeningHours\Exception\NotFoundException
+     * @throws \StadGent\Services\OpeningHours\Exception\ChannelNotFoundException
+     * @throws \StadGent\Services\OpeningHours\Exception\ServiceNotFoundException
+     */
+    public function openNow($serviceId, $channelId)
+    {
+        $cacheKey = $this->createChannelCacheKey(__FUNCTION__, $serviceId, $channelId);
+
+        // By default from cache.
+        $cached = $this->cacheGet($cacheKey);
```

Comment:
How long is this cached? The return value of open"Now" can change every minute.

Reply:
I will remove caching of the "OpenNow" response in the refactoring PR.